pywinauto/controls/atspi_controls.py

```diff
@@ -496,27 +496,27 @@ class ScrollBarWrapper(atspiwrapper.AtspiWrapper):
     def __init__(self, elem):
         """Initialize the control"""
         super(ScrollBarWrapper, self).__init__(elem)
-        self.value = self.element_info.get_value()
+        self.control_value = self.element_info.get_value()
```

Comment:
First I would name it `self.atspi_value_obj` to avoid confusion with text value as a search property.